### PR TITLE
Fixed theming codeblock, scrollbar and callout component

### PR DESCRIPTION
**Pull request by Devorein**

### CODE REVIEW

File: components/editor/BangleEditor.tsx [hunk 1]
```diff
@@ -77,13 +79,34 @@ export default function Editor({ markdown }: { markdown: string }) {
       tablePlugins(),
       columnResizing,
       floatingMenuPlugin(),
+      NodeView.createPlugin({
+        name: "blockquote",
+        containerDOM: ["blockquote"],
+        contentDOM: ["span"]
+      }),
+      NodeView.createPlugin({
```
Comment (mattcasey) [inline components/editor/BangleEditor.tsx:87]: do we still need codeBlock.plugins() and  blockquote.plugins() above?

File: components/editor/BangleEditor.tsx [hunk 2]
```diff
@@ -77,13 +79,34 @@ export default function Editor({ markdown }: { markdown: string }) {
...
   });
 
-  return <ReactBangleEditor state={state}>
+  return <ReactBangleEditor state={state} renderNodeViews={({ node, children }) => {
+    if (node.type.name === 'blockquote') {
+      return <BlockQuote>
+        {children}
+      </BlockQuote>;
+    } else if (node.type.name === "codeBlock") {
+      console.log(children)
```
Comment (mattcasey) [inline components/editor/BangleEditor.tsx:102]: console.log

**Review by mattcasey — APPROVED** [review]
approved with a question

File: components/editor/BangleEditor.tsx (hunk 1 above)
Comment (Devorein) [reply components/editor/BangleEditor.tsx:87]: yes, it doesn't work otherwise, unfortunately.

File: components/editor/BangleEditor.tsx (hunk 2 above)
Comment (Devorein) [reply components/editor/BangleEditor.tsx:102]: Nice catch

File: theme/styles.scss [hunk 3]
```diff
@@ -167,3 +162,272 @@ div#__next > div {
   font-weight: bold;
   text-align: left;
 }
+
```
Comment (Devorein) [inline theme/styles.scss:165]: I had to move the `style.css` from `@bangle.dev/core/style.css` as it overwrote our `blockquote` styling. End goal would be to completely gradually remove styling from this file and port them to styled components

Comment (mattcasey) [reply theme/styles.scss:165]: agreed

**Pull request merged**